Block/System/Config/Form/Extensions/Products.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © 2018 TechNWeb, Inc. All rights reserved.
+ * See TNW_LICENSE.txt for license details.
+ */
+namespace TNW\Marketing\Block\System\Config\Form\Extensions;
+
+use Magento\Backend\Block\Template\Context;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Module\ModuleList;
+
+class Products extends \Magento\Config\Block\System\Config\Form\Field
+{
+    /**
+     * Template path
+     *
+     * @var string
+     */
+    protected $_template = 'system/config/form/extensions/products.phtml';
+
+    /**
+     * Scope Config Interface
+     *
+     * @var ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    /**
+     * Installed module list
+     *
+     * @var array|string[]
+     */
+    private $moduleListArray;
+
+    /**
+     * Products constructor.
+     * @param Context $context
+     * @param ScopeConfigInterface $scopeConfig
+     * @param ModuleList $moduleList
+     * @param array $data
+     */
+    public function __construct(
+        Context $context,
+        ScopeConfigInterface $scopeConfig,
+        ModuleList $moduleList,
+        array $data = []
+    )
+    {
```

Comment:
When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them